test/xpu_api/random/device_tests/weibull_distribution_device_test.pass.cpp

```diff
@@ -24,9 +24,11 @@
 #include "common_for_device_tests.h"
 #endif // TEST_DPCPP_BACKEND_PRESENT && TEST_UNNAMED_LAMBDAS
 
+#if TEST_DPCPP_BACKEND_PRESENT && TEST_UNNAMED_LAMBDAS
```

Comment:
If looking more precisely all those variables can be moved under `#if TEST_DPCPP_BACKEND_PRESENT && TEST_UNNAMED_LAMBDAS`

Reply:
And I believe I have a lot of such places

Reply:
Fixed.

Reply:
That's not what I was asked for. I didn't ask to wrap them with macros. I asked to move them under already existing macros. Basically in `main` function under  
`#if TEST_DPCPP_BACKEND_PRESENT && TEST_UNNAMED_LAMBDAS`